### PR TITLE
Add .travis.yml

### PR DESCRIPTION
**Pull request by dbrgn**
Even if we don't have tests yet, we can at least test the build process :)

### CODE REVIEW

**Comment by rnestler** [comment]
Could you add a badge to the Readme?

**Comment by dbrgn** [comment]
I'm still trying to get the checks to run :)

**Comment by dbrgn** [comment]
Maybe you'll have to run `git submodule sync` on your instance.

**Comment by rnestler** [comment]
> Maybe you'll have to run git submodule sync on your instance.

Or git submodule init, or git submodule update. I always just type in every sudmodule command I know when having problems with submodules :wink:

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+rust:
+  - stable
+env:
+  - CRATE=service
+  - CRATE=validator
+before_script:
+  - |
+    python --version &&
+    pip --version &&
+    pip install 'travis-cargo<0.2' --user &&
```
Comment (rnestler) [inline .travis.yml]: Whats the advantage of travis-cargo?

Comment (dbrgn) [reply .travis.yml]: Hm, we probably don't need it in this case.

**Review by rnestler — APPROVED** [review]
Looks good now :+1:

**Pull request merged**